src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -336,7 +336,7 @@ record C(int X, int Y)
             var actualMembers = comp.GetMember<NamedTypeSymbol>("C").GetMembers().ToTestDisplayStrings();
             var expectedMembers = new[]
             {
-                "C C.<>Clone()",
+                "C C." + WellKnownMemberNames.CloneMethodName + "()",
```

Comment:
>." + WellKnownMemberNames.CloneMethodName + "()" [](start = 20, length = 48)

Please consider using "<Clone>$" explicitly.

Reply:
> Please consider using "&lt;Clone&gt;$" explicitly, here and throughout the tests.

I think this is going to create an unnecessary maintainability burden if we decide to rename the method in the future again. For each name I left a test that verifies its actual value. 

---
In reply to: [454516444](https://github.com/dotnet/roslyn/pull/45930#discussion_r454516444) [](ancestors = 454516444)

Reply:
You mean before we release? We can't rename these methods once we ship.

Reply:
(I guess we can the Clone one whose WellKnownMembers name is internal, not the others)